docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -29,7 +29,10 @@ As a core contributor it is necessary to have installed the following set of too
   be installed in a global location via the below command. Please see the installation
   docs on the cow repo for more setup details.
   `composer global require silverstripe/cow dev-master`
-* [transifex client](http://docs.transifex.com/client/). 
+* [satis repository tool](https://github.com/composer/satis). This should be installed
+  globally for minimum maintenance.
+  `composer global require composer/satis ^1@dev` (or `^1@alpha` for php 5)
```

Comment:
I don't actually think the `@alpha` works... at least alpha2 doesn't and I've opened an issue about it which was responded to with "use master not tags"

Also, 1.0.0 has now been tagged stable, so we probably shouldn't recommend `@dev`